h2o-logger/src/main/java/water/logging/Logger.java

```diff
@@ -0,0 +1,38 @@
+package water.logging;
+
+
+
+/**
+ * Logger provides SLF4J logging if SLF4J is on the classpath and otherwise provides logging straight to the console output.
+ */
+public class Logger <T extends Class> implements ILogger{
```

Comment:
Is seems this class is not needed any more and we can replace it with ILogger (it can be just the interface